tests/Fixer/PhpUnit/PhpUnitDataProviderNameFixerTest.php

```diff
@@ -441,5 +441,30 @@ public function f(): iterable {}
                 }
                 PHP,
         ];
+
+        yield 'with data provider as an attribute' => [
```

Comment:
Functional test for https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8225 should be added, then this PR looks complete as all other PRs are merged already.

Reply:
If you meant [this](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8197/commits/5bcf8020b04e75091dcfe82f52ae9c9fa9ce5216) unit test, then it's done.

Reply:
Yes and thanks!